faqs/risk-management.md

```diff
@@ -0,0 +1,116 @@
+
+## What is the Maker Foundation Interim Risk Team?
+
+The Maker Foundation Interim Risk Team is the first Risk Team whose role is to help bootstrap the Decentralized Risk Management function by providing an initial Risk Construct, as well as documentation, templates, and resources other Risk Teams can use.
+
+## How are Weightings for Risk Teams/Risk Constructs Determined?
+
+Weightings are determined by a number of factors, primarily the transparency of each team's Risk Construct, the accuracy of the work, rigorousness of the Risk Model and the general quality of the work as determined by MKR Token Holders. Higher transparency commands a higher weighting because it promotes more detailed and thorough audits.
+
+## Who chooses which Risk Constructs should be used as a basis for decision-making?
+
+Ultimately, MKR Holders decide on the quality of different Risk Constructs, and how they want to factor those Risk Constructs into their decision-making.
+
+When a risk team submits a new Risk Construct, MKR Holders must assess it and decide whether to vote-in the new Risk Team and with what weighting. The weighting for each Risk Team will be based on the overall quality and transparency of the submitted Risk Constructs.
+
+## What is the connection between a Risk Construct and voting in new collateral?
+
+Proposed new collateral will go through a due-diligence process, including a qualitative and quantitative assessment. Since every asset has differences, it is important to evaluate each of them individually according to a well-defined collateral assessment process. MKR Holders' weightings of these assessments will depend on which Risk Construct is used.  
+
+## What is Qualitative Risk Analysis, and how should one approach it?
+
+Qualitative Risk Analysis uses a relative or descriptive scale to measure risks based on their probability of occurring and the impact on the asset should they occur. To evaluate an asset for its risks, one needs to understand and assess the organization behind the asset, the characteristics of the token, and other relevant details. This is called a Qualitative assessment. The process of compiling the information has three parts:
+
+*   The Initial Collateral Onboarding Process: this covers the trade support structure, distribution of token holdings and available data series.
+
+*   The Operational Assessment Process: this covers the organization itself, the organization's operational risks, regulatory risks, the quality of the community, etc.
+
+*   The Technological Assessment Process: this covers the robustness and security of the underlying technology.
+
+Due diligence related to each of these three parts will give one information that can be compiled into distinct features and assigned ratings. These features are:
+
+*   Team; Core team and advisors
```

Comment:
Please bold definitions, use colons, and complete sentences. 

* **Team:** Core team and advisors.
* **Community:** Sentiment analysis.